src/cli.rs

```diff
@@ -183,28 +184,16 @@ pub(crate) fn policies(matches: &clap::ArgMatches) -> Result<HashMap<String, Pol
     })
 }
 
-pub(crate) fn verification_settings(matches: &clap::ArgMatches) -> Result<VerificationSettings> {
-    if matches.value_of("verification-path").is_none() {
-        return Err(anyhow!(
+pub(crate) fn verification_settings(
```

Comment:
Nit:

```suggestion
pub(crate) fn verification_config(
```
to match policy-fecther (and later variable)